applications/GeoMechanicsApplication/tests/cpp_tests/test_geomechanics_erosion_piping.cpp

```diff
@@ -0,0 +1,208 @@
+
+    p_piping_element = make_intrusive<SteadyStatePwPipingElement<2, 4>>(
+        3, CreateQuadrilateral2D4N(r_model_part, 9, 10, 11, 12, 1.0, 2.0, 1000.0, 1500.0),
+        p_piping_element_prop, std::make_unique<PlaneStrainStressState>());
+    p_piping_element->Initialize(r_process_info);
+    r_model_part.AddElement(p_piping_element);
+
+    Parameters p_parameters(R"(
+    {
+ 		"max_piping_iterations": 25
+    }  )");
+
+    const auto p_solver =
+        Kratos::make_shared<SkylineLUFactorizationSolver<SparseSpaceType, LocalSpaceType>>();
+    const auto p_scheme =
+        Kratos::make_shared<BackwardEulerQuasistaticPwScheme<SparseSpaceType, LocalSpaceType>>();
+    const auto p_builder_and_solver =
+        Kratos::make_shared<ResidualBasedBlockBuilderAndSolver<SparseSpaceType, LocalSpaceType, KratosExecute::LinearSolverType>>(
+            p_solver);
+    const KratosExecute::ConvergenceVariableListType convergence_settings{};
+    const auto p_criteria = std::make_shared<KratosExecute::MixedGenericCriteriaType>(convergence_settings);
+
+    auto p_solving_strategy =
+        Kratos::make_unique<MockGeoMechanicsNewtonRaphsonErosionProcessStrategy<SparseSpaceType, LocalSpaceType, KratosExecute::LinearSolverType>>(
+            r_model_part, p_scheme, p_solver, p_criteria, p_builder_and_solver, p_parameters);
+
+    return p_solving_strategy;
+}
+} // namespace Kratos
+
+namespace Kratos::Testing
+{
+KRATOS_TEST_CASE_IN_SUITE(CheckGetPipingElements, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Model model;
+    auto  p_solving_strategy = SetupPipingStrategy(model);
+
+    const auto piping_elements = p_solving_strategy->GetPipingElements();
+
+    constexpr int number_of_piping_elements = 3;
+
+    KRATOS_EXPECT_EQ(piping_elements.size(), number_of_piping_elements);
+    KRATOS_EXPECT_EQ(piping_elements[0]->GetId(), 1);
+    KRATOS_EXPECT_EQ(piping_elements[1]->GetId(), 3);
+    KRATOS_EXPECT_EQ(piping_elements[2]->GetId(), 2);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckFinalizeSolutionStep, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Model model;
+    auto  p_solving_strategy = SetupPipingStrategy(model);
+
+    p_solving_strategy->FinalizeSolutionStep();
+
+    auto elements = model.GetModelPart("ModelPart").Elements();
+
+    constexpr int active = 1;
+    KRATOS_EXPECT_EQ(elements[1].GetValue(PIPE_ACTIVE), active);
+
+    constexpr double expected_active_pipe_height = 0.0183333;
+    KRATOS_EXPECT_NEAR(elements[1].GetValue(PIPE_HEIGHT), expected_active_pipe_height, 1.0e-6);
+
+    constexpr int inactive = 0;
+    KRATOS_EXPECT_EQ(elements[2].GetValue(PIPE_ACTIVE), inactive);
+    KRATOS_EXPECT_EQ(elements[3].GetValue(PIPE_ACTIVE), inactive);
+
+    constexpr double expected_inactive_pipe_height = 0.0;
+    KRATOS_EXPECT_NEAR(elements[2].GetValue(PIPE_HEIGHT), expected_inactive_pipe_height, 1.0e-6);
+    KRATOS_EXPECT_NEAR(elements[3].GetValue(PIPE_HEIGHT), expected_inactive_pipe_height, 1.0e-6);
```

Comment:
We should be able to re-use the defaults we have in test_utilities
https://github.com/KratosMultiphysics/Kratos/blob/86e20cf7608d6c6371cfb755106617b17831e67a/applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities.h#L20-L26

Reply:
Thank you for the info. Changed.